src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -2185,6 +2185,14 @@ namespace OpenRCT2::Ui::Windows
             WindowSetResize(*this, kSizeRides, kSizeRides);
         }
 
+        static std::string GetRideString(RideId rideId)
+        {
+            const auto* ride = GetRide(rideId);
+            if (ride == nullptr)
+                return "";
```

Comment:
```suggestion
                return {};
```

I'd rather do this, so we always return an empty object, regardless of what the object is.

Reply:
For the changelog, would this be a fix or a feature?

Reply:
I don’t see the point, a string is a special type anyway.

Reply:
> For the changelog, would this be a fix or a feature?

A feature, I'd say, since the issue itself is an enhancement, not a bug.

> I don’t see the point, a string is a special type anyway.

It's just easier on refactors if they ever change return types, but ok leaving as is.

Reply:
> A feature, I'd say, since the issue itself is an enhancement, not a bug.

Personally, I'd say it's neither. It's relatively minor, so a `Change` in my book.

Reply:
Fair enough, too. I say @Gymnasiast has the final say given he opened the issue

Reply:
@AaronVanGeffen Size has very little do with whether something counts as an enhancement or change IMO. It’s more that a change is neutral to the user (or even slightly controversial), while an enhancement is an improvement to an existing feature.

> It's just easier on refactors if they ever change return types, but ok leaving as is.

I think it’s a good thing if changing the return type requires one to also modify this return statement. After all, it’s certainly not a given that the empty string from before still has the same meaning. I’m ultimately not too bothered though, but since it’s already `""`, let’s keep it that way.

Reply:
When returning an empty std::string it should use just default construction by either letting the compiler deduce the type or return `std::string{}`, constructing an empty std::string from `""` has a cost since std::string is not constexpr until C++23 or maybe even C++26.